tests/ttnn/unit_tests/operations/test_full.py

```diff
@@ -0,0 +1,113 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+import torch
+import torch.nn as nn
+import ttnn
+from models.utility_functions import comp_allclose
+from loguru import logger
+
+from tests.ttnn.utils_for_testing import assert_with_pcc, divup
+
+
+@pytest.mark.parametrize(
+    "input_shape",
+    [
+        [10],
+        [32, 32],  # single tile
+        [5, 17, 31],  # multiple tiles
+    ],
+)
+@pytest.mark.parametrize(
+    "fill_value",
+    [3, -1],
+)
+def test_full_int(device, input_shape, fill_value):
+    torch_any = torch.randint(0, 100, (input_shape), dtype=torch.int32)
+    torch_output_tensor = torch.full(input_shape, fill_value)
+
+    any = ttnn.from_torch(torch_any, layout=ttnn.TILE_LAYOUT)
+    any = ttnn.to_device(any, device)
+    output_tensor = ttnn.full_2(input_shape, fill_value, any)
+    assert ttnn.is_tensor_storage_on_device(output_tensor)
+    output_tensor = ttnn.from_device(output_tensor)
+    output_tensor = ttnn.to_torch(output_tensor)
+
+    passing, out = comp_allclose(torch_output_tensor, output_tensor, rtol=0.01, atol=0.01)
```

Comment:
In `full` op, `assert_equal` seems to be a better choice. Is there still a precision issue?

Reply:
I've updated